libs/ledger-live-common/src/families/cosmos/api/Cosmos.unit.test.ts

```diff
@@ -111,6 +111,99 @@ describe("CosmosApi", () => {
     });
   });
 
+  describe("getRedelegations", () => {
+    it("should return redelegations correctly (sdk v0.41/0.42/0.44/0.45)", async () => {
```

Comment:
The difference between the 2 tests (sdk version) is linked to the API response, correct?
If so, it could be great to use a variable name to better understand it :)

Reply:
Done :+1: